reference/functions/ifelse.markdown

```diff
@@ -89,3 +89,28 @@ bundle agent example
       "ifelse result list: $(mylist)";
 }
 ```
+
+**Note:** As a general rule function evaluation is skipped when undefined
+variables are used. However this function has special behavior when **exactly
+three** arguments are used allowing it to be actuated in order to enable the
```

Comment:
Instead of "actuated" I would use the same term you used above, i.e. "evaluated", to avoid confusion. I would say something like:
> However this function has special behaviour when **exactly three** arguments are used, it is being evaluated even if it contains undefined variables. For example: ...